remoter-client/src/wrapper.r

```diff
@@ -0,0 +1,100 @@
+require("RJSONIO")
+require("paws")
+require("zeallot")
+
+reload_from_s3 <- function(experiment_id) {
+    s3 <- paws::s3()
+    bucket_name <- paste("biomage-source", "staging", sep = "-")
+
+    c(body, ...rest) %<-% s3$get_object(
+        Bucket = bucket_name,
+        Key = paste(experiment_id, "r.rds", sep = "/")
+    )
+
+    obj <- readRDS(rawConnection(body))
+    return(obj)
+}
+
+
+run_step <- function(task_name, scdata, config) {
+    switch(task_name,
+        test_fn = {
+            import::from("test_fn.r", task)
+        },
+        stop(paste("Invalid task name given:", task_name))
+    )
+
+    out <- task(scdata, config)
+    return(out)
+}
+
+send_output_to_api <- function(input, output) {
+    c(config, plot_data = plotData) %<-% output
+
+    cluster_env <- Sys.getenv("CLUSTER_ENV", "staging")
+    sandbox_id <- Sys.getenv("SANDBOX_ID", "demo")
+    topic_name <- paste("work-results", cluster_env, sandbox_id, sep = "-")
```

Comment:
similarly, `work-results` is the name of the topic where we place the results and has to be saved in the configuration file and used from there